components/link/link.js

```diff
@@ -15,10 +16,15 @@ export const linkStyles = css`
 		bottom: 0.2rem;
 		top: 0.2rem;
 	}
-	.d2l-link:hover, .d2l-link:focus {
+	.d2l-link:hover {
 		color: var(--d2l-color-celestine-minus-1);
 		text-decoration: underline;
 	}
+	.d2l-link:focus {
+		outline: 2px solid var(--d2l-color-celestine);
+		outline-offset: 3px;
+		text-decoration: underline;
```

Comment:
Fun, this is a big change! We're good with the text colour staying as celestine on-focus but being celestine-minus-1 on hover?

Reply:
I think so, but I will double check with Jeff.

Reply:
Ok, confirmed with Jeff that this is expected.